### ISSUE TITLE
Muscle extensor surface location to skin lesions

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9993]
**Preferred term label:**
Muscle extensor surface location to skin lesions

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Erythema nodosum

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
is this distinct from
[Joint extensor surface localization HP:0032539](https://hpo.jax.org/app/browse/term/HP:0032539)?

**Comment by MickeySegal** [comment]
[Joint extensor surface localization HP:0032539](https://hpo.jax.org/app/browse/term/HP:0032539) refers to the joint.  The new term request refers to muscles.

**Comment by pnrobinson** [comment]
@MickeySegal 
I am not sure I understand what muscle extensor surface is, at least for many muscles. Do you have a definition or example?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK470369/ "Nodules appear on the extensor surfaces of the legs and knees, and sometimes thighs and forearms."

For the proximal upper extremity, this would be the surface over the triceps as opposed to the biceps.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000858
- label: Muscle extensor surface location
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: 
- def: Applies to an abnormality that is situated in extensor surface of an extremity.
- comment: For instance, in the upper extremity, this would refer to a lesion of the skin over the triceps as opposed to the biceps.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9993
- synonyms:
